sources/packages/backend/apps/queue-consumers/src/processors/schedulers/institution-integration/ier12-integration/_tests_/e2e/utils/string-utils.ts

```diff
@@ -14,6 +15,15 @@ export function numberToText(
   return value.toString().padStart(options?.length ?? 10, "0");
 }
 
+/**
```

Comment:
Is this method used anywhere?

Reply:
I am not sure if I am following the question. This method was added in the previous PR and it is used in 18 places.
![image](https://github.com/bcgov/SIMS/assets/61259237/f05c6b1c-afab-4e45-8f90-6a2f0410135b)

Reply:
got it